packages/neon/neon/lib/src/theme/neon.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+import 'package:neon/src/theme/branding.dart';
+import 'package:neon/src/theme/color_scheme.dart';
+import 'package:neon/src/theme/dialog.dart';
+import 'package:neon/src/widgets/dialog.dart';
+
+/// Defines the configuration of the overall visual [Theme] for a NeonApp
```

Comment:
```suggestion
/// Defines the configuration of the overall visual [Theme] for a [NeonApp]
```

Reply:
NeonApp does technically not make use of it and isn't exported from the neon package.

I rephrased it a bit.